tasks/connectors/qualys_was/qualys_was.rb

```diff
@@ -104,7 +109,7 @@ def run(opts)
         while more_records == true
           findings_response = qualys_was_get_webapp_findings(token, @options[:qualys_page_size].to_i, page)
           more_records = findings_response["ServiceResponse"]["hasMoreRecords"] == "true"
-          print_debug "there was a problem with the RESPONSE" if findings_response.nil?
+          fail_task "there was a problem with the RESPONSE. Nil value provided" if findings_response.nil?
```

Comment:
I think it's fine to fail if we're missing the entire page, but I would move it up one line. 111 will fail with an `NoMethodError: undefined method `[]' for nil:NilClass` error

Reply:
got it! thanks!